tools/pot/setup.py

```diff
@@ -65,21 +65,28 @@ def get_version():
     sys.argv.remove('--install-dev-extras')
 
 INSTALL_REQUIRES = [
-    "scipy>=1.5.4",
+    "scipy~=1.7; python_version == '3.7'",
+    "scipy>=1.8; python_version >= '3.8'",
     "jstyleson>=0.0.2",
     "numpy>=1.16.6,<=1.23.1",
     "addict>=2.4.0",
     "networkx~=2.5;python_version<='3.6'",
     "networkx<2.8.1;python_version>'3.6'",
     "tqdm>=4.54.1",
     "texttable>=1.6.3",
-    "pandas~=1.1.5",
+    "pandas~=1.3.5",
```

Comment:
in omz you use `pandas>=1.1.5,<=1.3.5` is there any reasons why it should be more strict for pot?

Reply:
We just bumped the dependency to checks, please sync separately with POT about this.

Reply:
I'm not understand why I need to sync if both files was updated by you :)

Reply:
If that's regarding https://github.com/openvinotoolkit/open_model_zoo/pull/3586, since OMZ is a separate project I did not want to break any compatibility OMZ might have with other packages, so extending the supported version range seems like a good idea. 

That was the motivation behind OMZ changes, the motivation behind POT changes is that the version was strict, so that's how we left it. Why exactly it's strict and not a range of versions is a question to POT.